sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -13,28 +13,31 @@ import com.mapbox.maps.viewannotation.ViewAnnotation.Companion.USER_FIXED_DIMENS
 import com.mapbox.maps.viewannotation.ViewAnnotationVisibility
 import java.util.concurrent.ConcurrentHashMap
 import java.util.concurrent.CopyOnWriteArraySet
-import kotlin.collections.HashMap
 
 internal class ViewAnnotationManagerImpl(
-  private val mapView: MapView
+  mapView: MapView,
+  private val viewAnnotationsLayout: FrameLayout = FrameLayout(mapView.context),
 ) : ViewAnnotationManager, ViewAnnotationPositionsUpdateListener {
 
   private val mapboxMap: MapboxMap = mapView.getMapboxMap()
-  private val viewPlugins = mapView.mapController.pluginRegistry.viewPlugins
   private val renderThread = mapView.mapController.renderer.renderThread
 
   init {
+    mapView.layoutParams = FrameLayout.LayoutParams(
+      ViewGroup.LayoutParams.MATCH_PARENT,
+      ViewGroup.LayoutParams.MATCH_PARENT,
+    )
+    // place the view annotations above the map (index 0) but below the compass, ruler and other plugin views
```

Comment:
Is it expected that view annotation should be placed below the compass/scalebar? Do we want to expose some configurations to adjust the z-order of the view annotations?

Reply:
that was not possible before that and in order to control Z index we have the concept of selected state for view annotatotions so it seems that we're totally OK here.

Reply:
@peng this kind of customisations makes sense to me, actually, though outside the scope of this PR I guess

Reply:
Dropped a ticket https://mapbox.atlassian.net/browse/MAPSAND-535